go/pkg/bertymessenger/service_test.go

```diff
@@ -432,18 +433,24 @@ func Test3PeersCreateJoinConversation(t *testing.T) {
 	defer cleanup()
 
 	// create nodes
+	joinersAmount := 2
+	accounts := make([]*TestingAccount, joinersAmount+1)
 	var creator *TestingAccount
 	{
 		creator = NewTestingAccount(ctx, t, clients[0], logger)
 		defer creator.Close()
+		accounts[0] = creator
 		creator.DrainInitEvents(t)
+		creator.SetNameAndDrainUpdate(t, "Creator")
 	}
-	var joiners = make([]*TestingAccount, 2)
+	var joiners = make([]*TestingAccount, joinersAmount)
 	{
-		for i := 0; i < 2; i++ {
+		for i := 0; i < joinersAmount; i++ {
 			joiners[i] = NewTestingAccount(ctx, t, clients[i+1], logger)
 			defer joiners[i].Close()
+			accounts[i+1] = joiners[i]
 			joiners[i].DrainInitEvents(t)
+			joiners[i].SetNameAndDrainUpdate(t, "Joiner #"+strconv.Itoa(i))
 		}
 	}
 
```

Comment:
you can simplify this init and make it more readable by keeping the old code and just do this at the end:

```go
accounts := append(joiners, creator)
```